index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Mobile First Project</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css"/>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+</head>
+<body>
+
+    <header>
+            <nav>
+                <ul class="header">
+                    <li class="time">9:41</li>
+                    <li class="cellular"><img src="./image/cellular.png" alt=""></li>
+                    <li class="wifi"><img src="./image/Wifi.png" alt=""></li>
+                    <li class="battery"><img src="./image/battery.png" alt=""></li>
+                   <li class="name">Paul Orife</li>
+                    <li class="menu"><i class="fas fa-bars"></i></li>
+                </ul>
+            </nav>
+    </header>
+
+    <main>
+          <section class="section1">
+            <div class="intro">
+                <h1>Hey There. <br>
+                    I'm Paul
+                </h1>
+            </div>
+
+            <div class="profession"><h2>I'm a Software Developer</h2></div>
+
+            <article class="services">
+                <p>
+                    I can help you build a product , feature or <br>
+                     website Look through some of my work <br>
+                      and experience! If you like what you see <br>
+                     and have a project you need coded, <br>
+                    don’t hestiate to contact me. <br>
+                </p>
+            </article>
+
+            <div>
+                <ul class="logo">
+                    <li><img src="./image/github.png" alt=""></li>
+                    <li><img src="./image/linkedln.png" alt=""></li>
+                    <li><img src="./image/angellist.png" alt=""></li>
+                    <li><img src="./image/twitter.png" alt=""></li>
+                    <li><img src="./image/madiu.png" alt=""></li>
+                </ul>
```

Comment:
- [ ] Imagine that I need to navigate to one of your social media by using these icons, once I click on it, oops nothing happens, yeah this is because you are not providing any link tag, I would suggest you wrap each of this in an `a` tag and additionally provide the correct link to your social media, please.